DigitalLearningSolutions.Web/Views/Shared/Components/SideMenuLinkWithHref/Default.cshtml

```diff
@@ -7,6 +7,8 @@
   </li>
 } else {
   <li class="side-nav__item">
-    <a class="side-nav__link" href="@Model.Href" asp-all-route-data="@Model.AspAllRouteData">@Model.LinkText</a>
+    <a class="side-nav__link" href="@Model.Href" asp-all-route-data="@Model.AspAllRouteData" target="@(Model.OpenInNewTab ? "_blank" : null)">

```

Comment:
Ah - we should probably add "noopener" to the true external links (GitHub):
https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

I wonder if it might be worth not reusing this component for these links in that case, so we can keep the component simple by not adding two extra properties just for two extra use cases.. Thoughts? 🤔

Reply:
Makes sense. Do we think there won't be many other side menus with links that need to open in new tabs then?

Reply:
Also, that link says it is not compatible with IE, I presume we are okay with that?

Reply:
Ah - could we also add noreferrer to handle IE (I think that's supported on newer IE versions)? 🙏  
Re side menus and new tabs - we're not expecting to have any other external links. If it ever changes, we can rethink whether to commonise them :)